docs/ops/sort/MulticlassNMS_8.md

```diff
@@ -0,0 +1,176 @@
+##  MulticlassNms<a name="MulticlassNms"></a> {#openvino_docs_ops_sort_MulticlassNms_8}
```

Comment:
Update `ie_docs.xml` file as in the https://github.com/openvinotoolkit/openvino/pull/5773